CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+CHANGES IN VERSION 1.10.0 (from 1.9.0)
+===================================
+
+Appium 1.10.0 is a minor release.
+
+#### General
+* Bring Espresso Driver out of beta and into general availability
```

Comment:
One more general improvement for both Android and iOS is in strings extraction area